### PR TITLE
Add more option to the contextual menu.

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

File: src/components/menus/directory-tree-contextual-menu.tsx [hunk 1]
```diff
@@ -48,13 +48,13 @@ interface DirectoryTreeContextualMenuProps {
     onClose: (e: unknown, nextSelectedDirectoryId?: string | null) => void;
     openDialog: string;
     setOpenDialog: (dialogId: string) => void;
-    isEmptyDirectory: boolean;
+    hideSomeMenuOptions: boolean;
```
Comment (jamal-khey) [inline src/components/menus/directory-tree-contextual-menu.tsx]: i would suggest to rename this props to 'isReadOnlyMode' or 'hideModifyDirectoryOptions"

**Review by jamal-khey — COMMENTED** [review]
i suggest to rename the props hideSomeMenuOptions

File: src/components/menus/directory-tree-contextual-menu.tsx (hunk 1 above)
Comment (AAJELLAL) [reply src/components/menus/directory-tree-contextual-menu.tsx]: I change it to restrictMenuItems.

**Review by jamal-khey — APPROVED** [review]
Test OK
Code Ok
no warn in the console

**Pull request merged**